components/automate-gateway/api/iam/v2beta/common/rules.proto

```diff
@@ -26,12 +26,19 @@ enum ConditionOperator {
     EQUALS = 2;
 }
 
+enum RuleStatus {
+    RULE_STATUS_UNSET = 0;
```

Comment:
I think an unset first enum is more important when it's **incoming** data, i.e. something some would `curl` at our API. But, I don't suppose it hurts.

Reply:
I only put it in here because `ListRules` isn't returning the status yet. Maybe we should just drop it from the gateway for now since we aren't using it.